Makefile

```diff
@@ -89,3 +90,9 @@ antora-preview: antora
 	@echo Preveiew should be available at http://localhost:5000
 	@echo ------------
 	podman run --rm -it -v $(CURDIR)/public:/usr/share/nginx/html:ro -p 5000:80 nginx
+
+build_container: Dockerfile
+	$(DOCKER) build . -t howto
+
+container:
+	$(DOCKER) run -v "$(CURDIR):/BUILD" -it howto sh -c 'make -C BUILD $(TARGET)'
```

Comment:
Not sure why you put TARGET variable there. I'll merge the PR and tweak it afterwards.

Reply:
There is a possibility of creating multiple container-* targets, but this would lead to code duplication. I think it is better to specify TARGET as a parameter so that we can build any target that the Makefile supports within Docker.

Reply:
But these multiple targets are not there yet. In short: YAGNI - don't write code until it is actually needed.